src/NosCore.GameObject/Character.cs

```diff
@@ -758,19 +761,19 @@ private NInvPacket BuyFrom(ShopItem item, short amount, short slotChar)
 
             if (Shop.ShopItems.Count == 0)
             {
-                CloseShop();
+                await CloseShop();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)